uk_election_timetables/elections/police_and_crime_commissioner.py

```diff
@@ -1,12 +1,12 @@
 from datetime import date
 
-from uk_election_timetables.calendars import working_days_before
+from uk_election_timetables.calendars import working_days_before, Country
 from uk_election_timetables.election import Election
 
 
 class PoliceAndCrimeCommissionerElection(Election):
     def __init__(self, poll_date: date):
-        self.poll_date = poll_date
+        Election.__init__(self, poll_date, Country.ENGLAND)
```

Comment:
I don't know if this matters for this use case, but PCCs are in both England and Wales, but not Scotland or Northern Ireland.

Reply:
The reason it might not matter in this case is that, AFIK, PCCs have all the same rules in England as they do in Wales

Reply:
Yeah, this is a weird case. I think in the near future a refactoring is due to separate the concept of `Country` (England, Wales, Scotland, Northern Ireland) and the yet-to-exist `CountryCalendar` (EnglandAndWales, Scotland, Northern Ireland).

As you identify, PCCs will only ever have one calendar applied so from the user POV this is absolutely fine, but the idea of a PCC in Wales being internally coded to be in England feels a bit wrong.

Reply:
In the meantime, I think _only_ elections that can vary by country in their dates (Local, UK Parl) accept countries as parameters. The rest override the `__init__` and supply their appropriate country.

Unless this is something particular to Python that i've misunderstood? (it's how I'd do it in Java!)